From 8c54eb691cbb6dde5677791e3178404da2df843f Mon Sep 17 00:00:00 2001
From: Christophe Priouzeau <christophe.priouzeau@linaro.org>
Date: Fri, 23 Jan 2015 10:09:45 +0100
Subject: [PATCH] OE: remove the compile choice

Signed-off-by: Christophe Priouzeau <christophe.priouzeau@linaro.org>
---
 host/Makefile | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/host/Makefile b/host/Makefile
index 2effc43..f740fca 100644
--- a/host/Makefile
+++ b/host/Makefile
@@ -1,11 +1,11 @@
 
-CC      = $(CROSS_COMPILE)gcc
-LD      = $(CROSS_COMPILE)ld
-AR      = $(CROSS_COMPILE)ar
-NM      = $(CROSS_COMPILE)nm
-OBJCOPY = $(CROSS_COMPILE)objcopy
-OBJDUMP = $(CROSS_COMPILE)objdump
-READELF = $(CROSS_COMPILE)readelf
+#CC      = $(CROSS_COMPILE)gcc
+#LD      = $(CROSS_COMPILE)ld
+#AR      = $(CROSS_COMPILE)ar
+#NM      = $(CROSS_COMPILE)nm
+#OBJCOPY = $(CROSS_COMPILE)objcopy
+#OBJDUMP = $(CROSS_COMPILE)objdump
+#READELF = $(CROSS_COMPILE)readelf
 
 OBJS = hello_world.o
 
-- 
1.9.1

