diff --git a/host/Makefile b/host/Makefile
index 2effc43..f740fca 100644
--- a/host/Makefile
+++ b/host/Makefile
@@ -1,11 +1,11 @@
 
-CC      = $(CROSS_COMPILE)gcc
-LD      = $(CROSS_COMPILE)ld
-AR      = $(CROSS_COMPILE)ar
-NM      = $(CROSS_COMPILE)nm
-OBJCOPY = $(CROSS_COMPILE)objcopy
-OBJDUMP = $(CROSS_COMPILE)objdump
-READELF = $(CROSS_COMPILE)readelf
+#CC      = $(CROSS_COMPILE)gcc
+#LD      = $(CROSS_COMPILE)ld
+#AR      = $(CROSS_COMPILE)ar
+#NM      = $(CROSS_COMPILE)nm
+#OBJCOPY = $(CROSS_COMPILE)objcopy
+#OBJDUMP = $(CROSS_COMPILE)objdump
+#READELF = $(CROSS_COMPILE)readelf
 
 OBJS = hello_world.o
 
diff --git a/ta/Makefile b/ta/Makefile
index 7c492b5..d45c71b 100644
--- a/ta/Makefile
+++ b/ta/Makefile
@@ -1,16 +1,5 @@
-CFG_TEE_TA_LOG_LEVEL ?= 2
+CFG_TEE_TA_LOG_LEVEL ?= 4
 CPPFLAGS += -DCFG_TEE_TA_LOG_LEVEL=$(CFG_TEE_TA_LOG_LEVEL)
 BINARY=8aaaf200-2450-11e4-abe20002a5d5c51b
 
 include $(TA_DEV_KIT_DIR)/mk/ta_dev_kit.mk
-
-all: $(BINARY).ta
-
-$(BINARY).ta: $(BINARY).bin
-	rm -f $@
-	cat faked_armv7_uta_signed_header.bin $< > $@
-
-clean: clean_ta_file
-.PHONY: clean_ta_file
-clean_ta_file:
-	rm -f $(BINARY).ta
